client/source/App.jsx

```diff
@@ -32,6 +33,8 @@ const App = () => {
       <Header />
       <Overview />
       <button onClick={testGetRequest}>TEST GET PRODUCTS REQUEST</button>
+      <RelatedList />
```

Comment:
You've got a merge conflict here. Nothing huge, but you should merge main into your branch and clear it up first.